DigitalLearningSolutions.Web/Controllers/LearningPortalController/SelfAssessment.cs

```diff
@@ -640,6 +653,12 @@ public IActionResult AddNewSupervisor(int selfAssessmentId)
                 selfAssessmentId,
                 User.GetUserIdKnownNotNull()
             );
+

```

Comment:
can we sort the supervisor list into alphabetical order here? I know it's not in the scope of the ticket but it would be a big improvement.

Reply:
supervisor list sorted by forename.